src/oauth-provider.ts

```diff
@@ -682,6 +689,19 @@ export interface Token {
   };
 }
 
+/**
+ * Token record with decrypted properties
+ * Derived from Token but with wrappedEncryptionKey removed and encryptedProps replaced with metadata
+ */
+export type TokenSummary<T = any> = Omit<Token, 'wrappedEncryptionKey' | 'grant'> & {
```

Comment:
Let's actually write out this type rather than declare it as `Token` with fields omitted... otherwise it's too easy to add something to `Token` that isn't appropriate for `TokenSummary` and not realize it will be exposed.